rest/authorization-server/src/main/java/org/eclipse/sw360/rest/authserver/security/Sw360SecurityEncryptor.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Siemens AG, 2018. Part of the SW360 Portal Project.
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.sw360.rest.authserver.security;
+
+import org.apache.log4j.Logger;
+
+import javax.crypto.*;
+import javax.crypto.spec.SecretKeySpec;
+import java.io.IOException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+
+public class Sw360SecurityEncryptor {
+
+    private static final Logger log = Logger.getLogger(Sw360SecurityEncryptor.class);
+
+    private static final byte[] SECRET_SPEC_KEY = "sw360-rest-super-secret-password".getBytes();
```

Comment:
might be better called `PUBLIC_SPEC_KEY` :wink:

Reply:
changed to `PUBLIC_SPEC_KEY`.
Your are right, its not really secret :stuck_out_tongue_winking_eye: